python/rapidsmpf/rapidsmpf/streaming/core/spillable_messages.pyi

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from rapidsmpf.buffer.content_description import ContentDescription
+from rapidsmpf.buffer.resource import BufferResource
+from rapidsmpf.streaming.core.message import Message
+
+class SpillableMessages:
+    def insert(self, message: Message) -> int: ...
+    def extract(self, mid: int) -> Message: ...
```

Comment:
```suggestion
    def insert(self, message: Message[Payload]) -> int: ...
    def extract(self, mid: int) -> Message[Payload]: ...
```

with appropriate import. Some typecheckers otherwise infer `Unknown` for the generic payload type.